src/main/java/org/profit/telegrambot/model/Product.java

```diff
@@ -0,0 +1,29 @@
+package org.profit.telegrambot.model;
+
+import lombok.*;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+public class Product {
+    private Integer id;
+    private Integer categoryId;
+    private String name;
+    private Double price;
+    private String image;
+    private boolean deleted = false;
```

Comment:
`boolean` has a default value of `false` 

Why do we need this variable predefine explicitly?